src/notebook/ipython_profiles/profile_narrative/kbase_templates/static/kbase/js/widgets/narrative_core/kbaseNarrativeMethodCell.js

```diff
@@ -10,7 +10,11 @@
  */
 
 (function( $, undefined ) {
-  require(['jquery', 'kbwidget'], function($) {
+  require(['jquery', 
+           'handlebars', 
+           'kbwidget', 
+           'kbaseAuthenticatedWidget',
+           'kbaseNarrativeCellMenu'], function($, Handlebars) {
```

Comment:
Why is the "Handlebars" parameter passed into the widget? This gives me an error locally because the Handlebars parameter overwrites the global Handlebars JS and is left undefined when this widget is created. If I just remove 'Handlebars' as a function parameter, it works as expected.

And sorry for not updating the require dependencies... I'm still getting used to using require in widgets, and figured it just wouldn't work if they aren't correct.

I also now realize that we need to add the kbaseAuthenticatedWidget dependency in kbaseNarrativeAppCell. I can do that in a separate PR.